src/pages/landing-page/landing-page.scss

```diff
@@ -0,0 +1,173 @@
+@import '../../assets/styles';
+@import '../../assets/colors';
+
+.landing-page {
+  width: 100%;
+  color: white;
+  background-color: $deck-cover-blue;
+
+  .full-viewport {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    min-height: $medium-screen;
+    height: 100vh;
+    padding: 0px 24px;
+    border-bottom: 4px solid $blue;
+
+    &:first-child {
+      height: calc(100vh - $header-height);
+    }
+  }
+
+  .scroll-down-anchor {
+    position: absolute;
+    bottom: 0;
+    left: 50%;
+    transform: translate(-50%, -50%);
+
+    @media screen and (max-height: $medium-screen) {
+      left: unset;
+      right: 0;
+    }
+
+    .scroll-down-icon {
+      fill: white;
+
+      height: 32px;
+      width: 32px;
+
+      &:hover {
+        @include standard-transition;
+        fill: $light-blue;
+      }
+    }
+  }
+
+  .text-right-align {
+    text-align: end;
+  }
+
+  .text-bold {
+    font-weight: bold;
+  }
+
+  .text-alternate {
+    color: $light-blue;
+    font-weight: bold;
+  }
+
+  .text-sm {
+    font-size: 16px;
+  }
+
+  .text-sm-md {
+    font-size: 24px;
+  }
+
+  .text-md {
+    font-size: 32px;
+  }
+
+  .text-md-lg {
+    font-size: 48px;
+  }
+
+  .text-lg {
+    font-size: 96px;
+  }
+
+  .text-sm-dynamic {
+    font-size: 1.1vw;
+  }
+
+  .text-md-dynamic {
+    font-size: 2.35vw;
+  }
+
+  .text-lg-dynamic {
+    font-size: min(12vw, 96px);
+  }
+
+  /**
+   * Page specific fonts begin here.
+   */
+  .top-page {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
+    padding: 48px 8%;
+
+    .heading-text {
+      position: relative;
+
+      .learn-anything {
+        position: absolute;
+        margin-top: 2vw;
+        right: 8px;
+        text-align: right;
+      }
+    }
+
+    .wave-image {
+      animation: pulse-animation 3s ease-in-out infinite alternate;
```

Comment:
I really like this pulse animation. It's pretty dope

Reply:
yay! I figure it's something we keep until we have a better idea of what we want to do w it